src/components/Admin/index.jsx

```diff
@@ -309,6 +316,13 @@ class Admin extends React.Component {
                   <h2>Overview</h2>
                 </div>
               </div>
+              <div className="row mt-4">
+                <div className="col">
```

Comment:
Can we use the Row and Col components available in [Paragon](https://paragon-openedx.netlify.app/foundations/layout/#row)?

Reply:
@jajjibhai008  I used Bootstrap's `row` and `col` because the original structure in Admin `index.jsx` used them. To keep things consistent, I continued with Bootstrap's `row` and `col` classes rather than switching to Paragon's `Row` and `Col` components.
<img width="877" alt="Screenshot 2023-10-10 at 12 29 16 PM" src="https://github.com/openedx/frontend-app-admin-portal/assets/113524403/743dbfe0-fdd2-49b9-96e0-6139049e3043">